cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -38,13 +38,13 @@ spec:
         - name: config-volume
           mountPath: /etc/google-fluentd/config.d
         env:
-        - name: STACKDRIVER_METADATA_AGENT_URL
-          value: {{ stackdriver_metadata_agent_url }}
         - name: NODE_NAME
           valueFrom:
             fieldRef:
               apiVersion: v1
               fieldPath: spec.nodeName
+        - name: STACKDRIVER_METADATA_AGENT_URL
+          value: http://$(NODE_NAME):8799
```

Comment:
Is this working? Bash substitution would be ${NODE_NAME}, as $(NODE_NAME) means "run NODE_NAME and put whatever it printed in the result".

Reply:
Yes, this works, http://jeffmendoza.github.io/kubernetes/v1.1/docs/design/expansion.html

Reply:
Cool, I wasn't aware of that, thanks!